app/javascript/listings/__tests__/classifiedFiltersCategories.test.jsx

```diff
@@ -0,0 +1,103 @@
+import { h } from 'preact';
+import { deep } from 'preact-render-spy';
+import ClassifiedFiltersCategories from '../components/ClassifiedFiltersCategories';
+
+describe('<ClassifiedFiltersCategories />', () => {
+  const firstCategory = {
```

Comment:
All the variables in the describe can be mutated by tests. If you need all of these for each test, change them to let and set them in a `beforeEach(() => {});`